client/server/pages/index.js

```diff
@@ -352,6 +352,16 @@ function getDefaultContext( request, entrypoint = 'entry-main' ) {
 		context.commitChecksum = getCurrentCommitShortChecksum();
 	}
 
+	if ( calypsoEnv === 'jetpack-cloud-stage' ) {
+		context.badge = 'jetpack-cloud-staging';
+		context.feedbackURL = 'https://github.com/Automattic/wp-calypso/issues/';
+	} else if ( calypsoEnv === 'jetpack-cloud-development' ) {
```

Comment:
Nit: seems like we use `else if` here and not for the other cases. Ideally, we would be consistent for all of those cases, or we would use a `switch` or something.

Reply:
Fixed in 24537a74f36310f46814241f30496a548db9ba3a. I debating refactoring, but decided it wasn't worth the energy for a micro optimization like that.